python/__init__.py

```diff
@@ -655,6 +649,10 @@ def set_uart_callback(self, uart, install):
   # Timeout is in ms. If set to 0, the timeout is infinite.
   CAN_SEND_TIMEOUT_MS = 10
 
+  @ensure_can_packet_version
```

Comment:
if we call reset on init, this won't allow old FW to connect

Reply:
not sure what you are suggesting?